testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -993,7 +993,7 @@ class TestUnwrapFlag(object):
             np.array([[7333.79167791, -211.8997285, -721.50785456],
                       [-211.8997285, 7059.07470427, -91.32156884],
                       [-721.50785456, -91.32156884, 6509.31735029]]),
-        'asphericity': 0.02060121,
+        'asphericity': np.array([0.135, 0.047, 0.094]),
```

Comment:
Why has this gone from a scalar to an array?

Reply:
As mentioned in https://github.com/MDAnalysis/mdanalysis/issues/3002, the shape and anisotropy functions produced one value no matter the chosen ``compound``. I implemented the coordinate handling use in the center of mass calculation (to handle ``pbc`` and ``wrap`` options with  the different ``compound ``options) in the new ``gyration_moments`` method which then also corrects this issue for these other metrics.

Reply:
Is there a selection option for `compound` with the new setup that would recapitulate this test value?

Reply:
Based on my read of issue #3002 and an understanding of what specifying the `compound` keyword is supposed to achieve, I think the proposed change offers the expected and desired result. 
I could add the initial code back in with an additional option `cluster=False` to say that the shape of the group should be assessed with compounds unwrapped beforehand.

Reply:
Right, I'm suggesting that I add it if you want to retain the previous output behavior, but the asphericity was so far from expected behavior (#3002), unless I'm misunderstanding the code, I'm not sure it's worth it since the "correct" combination of the keywords (i.e., using compound="group".) should reproduce this single value result, and if compound equals anything else, then it's wrong.

Old Asphericity steps:
1) Use `wrap`, `unwrap`, and `compound` to calculate an array/matrix of the center of masses
2) If compound != "group", combine the centers of mass into a single center of mass
    - Effectively obtaining the result for compound="group" but the expression multiplies the atomic mass (size n_atoms) but the com array (n_residues or n_segments) and then adds them to produce a single COM, so I'm not sure how that works.
3) Use single COM to calculate a single asphericity parameter

Unless I'm misunderstanding, steps 2 and 3 are not the desired behavior when compound != "group".